samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+.. _wifi_throughput_sample:
+
+Wi-Fi: Throughput
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Throughput sample demonstrates how to measure the throughputs for Nordic Semiconductor's Wi-Fi® chipsets.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample demonstrates how to measure network performance using **zperf** tool in different profiles.
+Using this sample, the development kit can associate with an access point.
```

Comment:
What is the real purpose of adding this sentence?
If I understand correctly, the point should be that it can then receive and send networking traffic which allows users to measure its throughput".
But I could also skip this completely.